helm/cluster-azure/templates/helmreleases/network-policies.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

Comment:
This is already part of the cluster chart ([see here](https://github.com/giantswarm/cluster/blob/main/helm/cluster/templates/apps/network-policies.yaml)), so we must not add this to provider-specific charts one by one.

Let's make this consistent with `cluster-aws` (except that you may not need provider-specific settings?? [example in cluster-aws which has some extra stuff for proxy-private clusters](https://github.com/giantswarm/cluster-aws/blob/67995d3415cad36a6d0e0d55e70a51037622ba81/helm/cluster-aws/templates/_network-policies_helmrelease_config.yaml)).

Reply:
Yes thank you. I'm aware. This PR should be a draft; sorry for the noise.